.github/workflows/arginfo-files.yml

```diff
@@ -0,0 +1,45 @@
+name: "Check generated arginfo files"
+
+on:
+  pull_request:
+    branches:
+      - "v*.*"
+      - "master"
+      - "feature/*"
```

Comment:
This was added in https://github.com/mongodb/mongo-php-driver/commit/69cc2b5c7418ef4bf320a160b3d6b0918ef7102d, and it's a nice way to prevent builds on all branches while retaining the ability to test feature branches in our forks without messing with the workflow files.

Reply:
Oh, I forgot I added that. IIRC, this was back when we had a separate branch for SDAM monitoring.